convert/jenkinsjson/json/jira_test.go

```diff
@@ -0,0 +1,144 @@
+package json
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestConvertJiraBuild(t *testing.T) {
```

Comment:
these tests are not doing anything. Please refer to something like this: convert/jenkinsjson/json/findFiles_test.go